### ISSUE TITLE
Failed to connect with the camera (NVR)

### ISSUE THREAD

**Issue by MacL3an** [fwestenberg/reolink_dev#302]
**Describe the bug**
I cannot connect to my Reolink NVR.

**To Reproduce**
I have installed version 2.6 using HACS.
I go Configuration, Integration, Add Integration, Find `Reolink IP Camera` and fill in the IP address of the NVR and use port 80. I get the error `Failed to connect with the camera`.
The error logs say:
```
2021-10-08 20:35:06 DEBUG (MainThread) [reolink.camera_api] Reolink camera with host http://10.0.1.15/:80 trying to login with user hass
2021-10-08 20:35:11 DEBUG (MainThread) [reolink.camera_api] Host http://10.0.1.15/: Connection error Cannot connect to host http:80 ssl:default [Try again]
2021-10-08 20:35:11 DEBUG (MainThread) [reolink.camera_api] Host http://10.0.1.15/: Error translating Reolink settings response
```

**Screenshots**
<img width="835" alt="image" src="https://user-images.githubusercontent.com/2429736/136609279-cce6aeb6-a279-4438-a28b-5bde0bba0184.png">

**Environment:**
Please provide useful information about your environment, like:
 - Home Assistant version: Reolink Dev 0.26
 - Reolink camera model: NVR RLN8-410
 - Camera firmware number: NVR firmware 3.0.0.130

**Additional context**
If I connect directly to the camera via HASS config it works:
```
camera:
  - platform: generic
    name: Cam0
    still_image_url: "http://10.0.1.15/cgi-bin/api.cgi?cmd=Snap&amp;channel=0&amp;rs=iamarandomstringoflettersand12345&amp;user=XXX&amp;password=YYYY"
    stream_source: "rtsp://XXX:YYY@10.0.1.15:554/h264Preview_01_sub"
```

**Comment by markaggar** [comment]
I upgraded from v22 (which did work, although motion sensors do not update reliably) to v26, and I'm getting this message

Logger: custom_components.reolink_dev.base
Source: custom_components/reolink_dev/base.py:360
Integration: Reolink IP camera (documentation, issues)
First occurred: 8:21:10 AM (630 occurrences)
Last logged: 5:49:14 PM

Host 10.0.0.195 error renewing the Reolink subscription

**Comment by MacL3an** [comment]
@markaggar, fantastic it worked! thanks! When this integration works, does it give any more functionality or does it just automate this setup?

Then a question from a HASS beginner: How do I create an automation when a motion is triggered? I assume I use the `State` trigger, but how do I figure out what to write in `From` and `To`? I've looked in the history for the entity, but then I only se "human friendly" names which I assume do not work and I don't see anything in the Supervisor logs, but I'm probably looking in the wrong place.

![image](https://user-images.githubusercontent.com/2429736/136672724-ebc52ff5-299d-44ea-b45c-a5acb9704b9a.png)

**Comment by markaggar** [comment]
You can literally write "on" (without the quotes) in the to: field.  Has to be lower case.  The integration (when it works) does let you control various states such as IR mode, push notifications and recording.

**Comment by MacL3an** [comment]
I just figured it out myself by setting the logging level at `DEBUG` and then going to Configuration/Logs and I found this:
```
2021-10-09 22:30:28 DEBUG (MainThread) [homeassistant.components.websocket_api.http.connection] [547809385104] Sending {"id": 2, "type": "event", "event": {"event_type": "state_changed", "data": {"entity_id": "binary_sensor.cam01_is_people", "old_state": {"entity_id": "binary_sensor.cam01_is_people", "state": "on", "attributes": {"friendly_name": "cam01_is_people", "device_class": "motion"}, "last_changed": "2021-10-09T20:30:20.024422+00:00", "last_updated": "2021-10-09T20:30:20.024422+00:00", "context": {"id": "de7e4874482a98f98f417c96830a86b5", "parent_id": null, "user_id": null}}, "new_state": {"entity_id": "binary_sensor.cam01_is_people", "state": "off", "attributes": {"friendly_name": "cam01_is_people", "device_class": "motion"}, "last_changed": "2021-10-09T20:30:28.018783+00:00", "last_updated": "2021-10-09T20:30:28.018783+00:00", "context": {"id": "1f111743ad739788b3ab5b20f63db2e7", "parent_id": null, "user_id": null}}}, "origin": "LOCAL", "time_fired": "2021-10-09T20:30:28.018783+00:00", "context": {"id": "1f111743ad739788b3ab5b20f63db2e7", "parent_id": null, "user_id": null}}}
```

But thanks for taking the time to write. Since it is a 'binary' sensor I guess on/off makes sense :). 

States such as IR mode and push can be controlled via the reolink app, so I guess I'm not loosing any functionality then. Thanks again for your help!

**Comment by markaggar** [comment]
You can also go to the developer tools section and see what the current state is for entities.

**Comment by cpainchaud** [comment]
@MacL3an it clearly states it cannot connect to 10.0.1.15 on port 8000 at the time, did you check that port was open on your camera around the same time (this is used by ONVIF protocol)

**Comment by MacL3an** [comment]
All I know is that in the reolink app the port for ONVIF is set to 8000 (the default) in the NVR:
![image](https://user-images.githubusercontent.com/2429736/136925625-294a8971-210c-4bff-b00e-f9eff5138247.png)

I'm not sure if there is any other setting I can play around with or something else I can investigate? It can't be a setting on my router, right?

**Comment by cpainchaud** [comment]
Onvif and Rtsp are 2 different service. Onvif seems to be disabled on yours

**Comment by hiraniha** [comment]
I also did this same thing yesterday,   I made the changes to the NVR to enable Onvif Protocol, then restarted the NVR

Then in HA I already had the cameras added from a previous version v22.  I had to remove them all after installing v26, then added each channel using my NVR ip address back in.  If I didn't remove them and add them back in the upgrade also didn't work properly, all the motion just showed unavailable.

Once I did that the separate motion sensors appeared as devices and are now working

![image](https://user-images.githubusercontent.com/73940881/137044647-4975937f-5300-4290-b5ca-9f59eb52767b.png)
